plugins/processors/cumulative_sum/sample.conf

```
@@ -0,0 +1,8 @@
+# Compute the cumulative sum of the given fields
+[[processors.cumulative_sum]]
+  ## Numerical fields to be processed (accepting wildcards)
+  # fields = ["*"]
+
+  ## Interval after which the sum value is reset to zero
+  ## If zero or unset the sum is never reset.
+  # expiry_interval = "0s"
```

I feel like we need to add some more words here... How about
```suggestion
  ## Interval after which metrics are evicted from the cache and the
  ## sum values are reset to zero. A zero or unset value will keep the
  ## metric forever.
  ## It is strongly recommended to set an expiry interval to avoid 
  ## growing memory usage when varying metric series are processed.
  # expiry_interval = "0s"

```